src/Command/SelfUpdateCommand.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+/*
+ * This file is part of the Manalize project.
+ *
+ * (c) Manala <contact@manala.io>
+ *
+ * For the full copyright and license information, please refer to the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Manala\Manalize\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * Manala self-update command.
+ *
+ * Mostly taken from https://github.com/FriendsOfPhp/PHP-CS-FIXER licensed under MIT
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * @author Robin Chalas <robin.chalas@gmail.com>
+ */
+class SelfUpdateCommand extends Command
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('self-update')
+            ->setAliases(['selfupdate'])
+            ->setDescription('Updates manalize to the latest stable version.')
+        ;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $io = new SymfonyStyle($input, $output);
+
+        if (!$this->isPharInstallation()) {
+            $io->writeln('<error>Self-update is available only for PHAR installation.</error>');
+
+            return 1;
+        }
+
+        $lastTag = $this->getLastTag();
+
+        if (null === $lastTag) {
+            $io->writeln('<error>Unable to determine newest version.</error>');
+
+            return;
+        }
+
+        if ('v'.$this->getApplication()->getVersion() === $lastTag) {
+            $io->writeln(sprintf('<info>manalize is already up to date (%s).</info>', $lastTag));
```

Comment:
Can be considered as a success IMO (`$io->success("Manalize is already up to date ($lastTag)")`).